samples/apps/basic/basic.cpp

```diff
@@ -19,7 +19,7 @@ using namespace nlohmann;
 namespace basicapp
 {
   using RecordsMap = kv::Map<std::string, std::vector<uint8_t>>;
-  static constexpr auto PRIVATE_RECORDS = "records";
+  static constexpr auto PRIVATE_RECORDS = "basic.records";
```

Comment:
"Why?"

Because I want to install the JS implementation of the logging app on this (the C++ basic app). But both of them try to use a map called `records`! You can add a JS endpoint called `POST /log/private` which tries to write to `records`, and install it via basic's `PUT /custom_endpoints`, but then when you _call_ `POST /log/private` the new map restrictions (#6240) say "hang on, this JS code can't access my C++ controlled table!". Cheap fix is to change the map name that basic wants to reserve, to not collide with the logging app.